stable/helm-exporter/templates/service.yaml

```diff
@@ -15,7 +15,7 @@ spec:
   type: ClusterIP
   clusterIP: None
   ports:
-    - port: 9100
```

Comment:
You likely need to modify the deployment.yaml port also

Reply:
Thanks for the highlight. Fixed it!